### PR TITLE
Now should work with InfluxDB 0.10

### PR DESCRIPTION
**Pull request by svazzole**
This should fix the problems with influx 0.10 @beckettsean

### CODE REVIEW

**Comment by beckettsean** [comment]
@jcheng5 we need eyes on this from someone who groks R. Do you have a moment to take a look?

File: R/influxdb.R [hunk 1]
```diff
@@ -6,52 +6,52 @@ NULL
...
+      colNames <- unlist(response$columns)
+      nr <- length(response$values)
+      
+      # build the dataframe
+      for (i in 1:nr) {
+        r <- t(unlist(response$values[[i]]))
+        if (i == 1) {
+          output <- data.frame(r, stringsAsFactors = FALSE)
+        } else {
+          output <- rbind(output, data.frame(r))
```
Comment (jcheng5) [inline R/influxdb.R]: (Did the influxdb response format drastically change?)

This is likely an extremely slow way to build up a dataframe (assuming row count >> col count) as every rbind call makes a copy of all the data so far. You'd have to experiment to find the fastest incantation but it usually involves mapping over an entire list using lapply/vapply (as the output vector will then be allocated just once), and then construct the data frame in one step.

File: R/influxdb.R [hunk 2]
```diff
@@ -6,52 +6,52 @@ NULL
...
-      u = username,
-      p = password,
-      q = query,
-      time_precision = match.arg(time_precision),
-      chunked = "false"
-    )
-  )
+influxdb_query <- function (host, port, database, query) {
+  
+  url <- paste0(host, ":", port, "/query?")
```
Comment (jcheng5) [inline R/influxdb.R]: Is this `?` needed? I would've thought `httr::GET` would add it.

Comment (jcheng5) [inline R/influxdb.R]: What happened to the username/password? Not needed anymore?

**Pull request closed without merging**